api-operator/deploy/controller-configs/controller_conf.yaml

```diff
@@ -21,9 +21,9 @@ metadata:
   namespace: wso2-system
 data:
   #mgw toolkit image to initialize/setup the micro gw project
-  mgwToolkitImg: wso2am/wso2micro-gw-toolkit:3.2.0-beta
+  mgwToolkitImg: rakhitharr/wso2micro-gw-toolkit:3.2.0-custom1
   #mgw runtime image to use in the mgw executable
-  mgwRuntimeImg: wso2/wso2micro-gw:3.2.0-beta
+  mgwRuntimeImg: rakhitharr/wso2micro-gw:3.2.0-custom1
```

Comment:
Shall we remove the user-specific location?

Reply:
Those were added only for pre release testing purposes. Removed in the latest code.